cli_reference/basic_cli_operations.adoc

```diff
@@ -117,6 +118,7 @@ syntax:
 |`pod` |`po`
 |`ResourceQuota` | `quota`
 |`replicationController` |`rc`
+|`replicaSet` (Technology Preview)|`rs`
```

Comment:
Ah I see replica set listed as Tech Preview in Origin 1.3 here:

https://github.com/openshift/origin#alpha-and-unsupported-kubernetes-features